sklearn/preprocessing/_encoders.py

```diff
@@ -323,7 +343,7 @@ def _compute_drop_idx(self):
                              zip(self.drop, self.categories_)], dtype=np.int_)
         else:
```

Comment:
I this this else be removed and you can replace the last elif with an else

Reply:
Done